stdlib/LibGit2/src/remote.jl

```diff
@@ -133,9 +133,11 @@ julia> LibGit2.url(remote)
 """
 function url(rmt::GitRemote)
     ensure_initialized()
-    url_ptr = ccall((:git_remote_url, libgit2), Cstring, (Ptr{Cvoid},), rmt.ptr)
-    url_ptr == C_NULL && return ""
-    return unsafe_string(url_ptr)
+    GC.@preserve rmt begin
```

Comment:
Same here

Reply:
For similar reasons as above, the `rmt` object is preserved until the return of `unsafe_string`.